api/envoy/api/v2/auth/cert.proto

```diff
@@ -279,6 +292,10 @@ message CertificateValidationContext {
 
   // If specified, Envoy will not reject expired certificates.
   bool allow_expired_certificate = 8;
+
+  // Certificate trust chain verification mode.
```

Comment:
This seem highly HTTP centric (at least from the description above), but will also apply to TCP proxying and so on. Can you clarify in comments?

Reply:
I've updated the comments, let me know if they need further tweaking